gateway/ln-gateway/src/cln.rs

```diff
@@ -33,24 +33,18 @@ where
 // See: https://github.com/ElementsProject/lightning/blob/master/doc/PLUGINS.md#htlc_accepted
 #[derive(Clone, Serialize, Deserialize, Debug)]
 pub struct Htlc {
-    pub short_channel_id: String,
     #[serde(deserialize_with = "as_fedimint_amount")]
     pub amount_msat: Amount,
+    // TODO: use these to validate we can actually redeem the HTLC in time
     pub cltv_expiry: u32,
```

Comment:
@okjodom We need to verify that we have enough time to buy the preimage by checking the CLTV expiry here. I don't understand the details, it's probably best to read the BOLTs and talk to LDK.

Reply:
ack.